ui/app/templates/rule_modal.html

```diff
@@ -47,15 +47,31 @@ <h3 class="modal-title" ng-show="!is_edit && is_duplicate">Duplicate Existing Ru
       <div class="col-md-6">
         <div class="form-group" ng-class="{'has-error': errors.backgroundRate}">
           <label for="id_rate">Rate</label>
-          <input type="text" class="form-control" id="id_rate" ng-model="rule.backgroundRate">
-          <p class="help-block" ng-show="errors.backgroundRate">{{ errors.backgroundRate.join(', ') }}</p>
+          <input type="text"
+                 class="form-control"
+                 ng-class="{'validation_errors': integer_validation_errors.rate}"
+                 name="rate"
+                 id="id_rate"
+                 ng-model="rule.backgroundRate"
+                 required>
+          <div class="form-group" ng-class="{'has-error': integer_validation_errors.rate}" ng-show="integer_validation_errors.rate">
```

Comment:
I think there's a subtle bug here by not looking at errors.backgroundRate - if the backend catches an issue with this field that the frontend doesn't, the error won't be displayed. Can you make sure we still show backend errors, too?

Reply:
I have highlighted this in another comment, but gist is the backend validation errors are displayed in the second-to-last error label div. This includes the ones caught for the *backgroundRate* and *Priority*. 
<img width="1113" alt="screen shot 2017-10-04 at 11 23 04" src="https://user-images.githubusercontent.com/28535137/31170154-b85ea8fa-a903-11e7-9444-c6d6252fe1ee.png">

Let me see if I can decode the `details` in the response object and have these error labels also light up should an error be caught by the backend validators. But in view, this in as much brings about uniformity in the error display, it will also be duplication of error flagging on the UI.

Reply:
It's OK not to worry about changing how the backend errors are displayed in this PR.